tests/e2e/workloads/app/test_jenkins_simulation.py

```diff
@@ -54,7 +54,7 @@ def test_git_clone(self, pod):
         relevant_pod_logs = None
         func_calls = ["NodeStageVolume", "NodeGetVolumeStats"]
         error_msg = "System has not been booted with systemd"
-
+        inode_info = '"unit":2'
```

Comment:
@avd-sagare how this test is relevent to the steps mentioned in here https://bugzilla.redhat.com/show_bug.cgi?id=2132270#c8 ?

The verification steps: 

1) Create CephFS pvc
2) Create a pod and attach the pod to a pvc 
3) Check on which nodes the pods are running

These steps are missing!!(?)

Reply:
These steps are executing as part of setup.

Reply:
ack. I see step 3 you're not performing properly here. you need to first identify the node where your pod is running then fetch the csi-cephfs-plugin pod instances which are running on the same node. Here you're doing it otherway round!

Reply:
check line number 60 -67 . Its already there.  # Get the node running this pod and #Get the csi_cephfsplugin pod running on this node. I hope I am not wrong